cmd/core-common-config-bootstrapper/res/configuration.yaml

```diff
@@ -26,6 +26,7 @@ all-services:
     MaxResultCount: 1024
     MaxRequestSize: 0 # Not currently used. Defines the maximum size of http request body in bytes
     RequestTimeout: "5s"
+    EnableNameFieldEscape: false # The name field escape could allow the system to use special or Chinese characters in the different name fields, including device, profile, and so on.  If the EnableNameFieldEscape is false, some special characters might cause system error.
```

Comment:
Add to comment that it is set to false by default to avoid breaking change and will be removed in EdgeX 4.0

Reply:
Added the comment and updated the commit message.